src/jsrs_parser.cc

```diff
@@ -524,25 +545,21 @@ Local<String> ParseKeyInObject(Isolate*    isolate,
   } else {
     size_t current_length = 0;
     for (size_t i = 0; i < *size; i++) {
-      if (begin[i] == ':') {
+      if (begin[i] == '_' || (i != 0 ?
+                              isalnum(begin[i]) :
+                              isalpha(begin[i]))) {
+        current_length++;
+      } else {
         if (current_length != 0) {
           result = String::NewFromUtf8(isolate, begin,
                                        NewStringType::kInternalized,
                                        static_cast<int>(current_length))
                                            .ToLocalChecked();
           break;
         } else {
-          THROW_EXCEPTION(SyntaxError, "Unexpected token :");
+          THROW_EXCEPTION(SyntaxError, "Unexpected identifier");
           return Local<String>();
```

Comment:
I'd like it to be consistent. In some cases we use `Local<String>`, in some cases — `String::Empty`. Though it is the matter of a different PR since it was not introduced in this one.